.arclint

```diff
@@ -2,7 +2,7 @@
   "linters": {
     "katex-linter": {
       "type": "script-and-regex",
-      "script-and-regex.script": "make lint || true",
+      "script-and-regex.script": "npm run lint || true",
```

Comment:
Thanks for updating this file.  We use phabricator/arcanist internally, but I can't remember the last time I saw a phabricator review for KaTeX.